### ISSUE TITLE
Tags and releases

### ISSUE THREAD

**Issue by ehsavoie** [ai-for-java/openai4j#48]
It would be nice to create tags on new releases and mybe add them to Github releases as it helps a lot with finding out what has changed :)

**Comment by langchain4j** [comment]
Right, I did not do that for a while. BTW I plan to move the code from this repository to LC4j repo soon, so this won't be needed.
